src/main/java/org/seqdoop/hadoop_bam/VCFFormat.java

```diff
@@ -59,4 +61,9 @@ public static VCFFormat inferFromData(final InputStream in) throws IOException {
 		}
 		return null;
 	}
+
+	public static boolean isBGZF(final InputStream in) throws IOException {
```

Comment:
This seems a bit out of place to me, since its not really VCF-specific, and there are other call sites that access BlockCompressedInputStream.isValidFile directly. It seems like we could get rid of this and the corresponding tests, since its just a pass-through. If you do want to keep it though, we should probably change BCFSplitGuesser to call it rather than calling isValidFile directly.